packages/webapp/components/layouts/AccountLayout/index.tsx

```diff
@@ -68,50 +80,25 @@ export default function AccountLayout({
     );
   }
 
-  if (!profile || !Object.keys(profile).length) {
-    return null;
-  }
-
-  const Seo: NextSeoProps = profile
-    ? {
-        title: getTemplatedTitle(profile.name),
-        description: profile.bio
-          ? profile.bio
-          : `Check out ${profile.name}'s profile`,
-        openGraph: {
-          images: [{ url: profile.image }],
-        },
-        twitter: {
-          handle: profile.twitter,
-        },
-      }
-    : {};
```

Comment:
This SEO template won't make sense here since this is the account pages. We don't do any form of SSR here, nor is the content dynamic based on some user id. This seems to be copy-pasted from the `/[userId]` page.